tests/cacl/test_cacl_application.py

```diff
@@ -5,7 +5,8 @@
 
 from tests.common.config_reload import config_reload
 from tests.common.utilities import wait_until
-
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports_to_upper_tor
```

Comment:
It would add some extra cacl in standby tor for standby port, if that is the case, should we add one more test case to cover this other than randomly pick up a ToR to run the test?

Reply:
> It would add some extra cacl in standby tor for standby port, if that is the case, should we add one more test case to cover this other than randomly pick up a ToR to run the test?

@StormLiangMS  I agree with you, add a new test case `test_cacl_application_dualtor` in my latest commit. Please review, thanks.